src/account/Account.java

```diff
@@ -0,0 +1,13 @@
+package account;
+
+public class Account {
+	public int accountNumber;
+	public String accountHoldersName;
+	public double accountBalance;
+	
```

Comment:
we need createdDate, accountType, userId
we need to create a class for hoding AccountHistory data
that class can hold different data about AccountHistory